docs/legal/CREDITS.md

```diff
@@ -67,33 +67,32 @@ We would like to credit other super sweet tools/frameworks that aid in the devel
 
 ## Third Party Licenses - Runtime
 
-Chocolatey CLI open source uses a number of 3rd party components. Their details are below (order is alphabetical).
-
-<!-- TOC -->
-
-- [7-Zip](#7-zip)
-- [AlphaFS](#alphafs)
-- [Checksum](#checksum)
-- [log4net](#log4net)
-- [Microsoft.Web.Xdt](#microsoftwebxdt)
-- [NuGet.Core (modified)](#nugetcore-modified)
-- [Rhino.Licensing (modified)](#rhinolicensing-modified)
-- [Rx (Reactive Extensions)](#rx-reactive-extensions)
-- [Shim Generator (shimgen)](#shim-generator-shimgen)
-- [SimpleInjector](#simpleinjector)
-
-<!-- /TOC -->
-
-### 7-Zip
-Chocolatey uses [7-Zip](http://www.7-zip.org/) for uncompressing archives. [License terms](http://www.7-zip.org/license.txt):
-
-~~~
+Chocolatey CLI (choco) uses a number of 3rd-party components.
+Their details are below.
+
+- [7-Zip @ 21.07](#7-zip--2107)
+- [AlphaFS @ 2.1.3](#alphafs--213)
+- [Checksum @ 0.2.0](#checksum--020)
+- [log4net @ 2.0.12](#log4net--2012)
+- [Microsoft.Web.Xdt @ 2.1.1](#microsoftwebxdt--211)
+- [NuGet.Core @ 2.11.0.20211014 (modified)](#nugetcore--211020211014-modified)
+- [Rhino.Licensing @ 1.4.1 (modified)](#rhinolicensing--141-modified)
+- [Rx (Reactive Extensions) @ 2.1.30214.0](#rx-reactive-extensions--21302140)
+- [Shim Generator (shimgen) @ 1.0.0](#shim-generator-shimgen--100)
+- [SimpleInjector @ 2.5.0](#simpleinjector--250)

```

Comment:
SimpleInjector should be `2.8.3`. It was updated as a part of the command registration PR.

Reply:
@TheCakeIsNaOH thanks for catching this! I have fixed this up!